pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -156,8 +156,20 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 			},
 			Rules: []authorizationapi.PolicyRule{
```

Comment:
This just in: throw in `list` for pods as well, since I'll be switching to a reflector/FIFO setup for the pod watch functionality.